bot.py

```diff
@@ -112,6 +112,64 @@ async def list(ctx):
     )
 
 
+@bot.command()
+async def add(ctx):
+    author = ctx.message.author.id
+    await ctx.message.channel.send(
+        embed = Embed.from_dict(
```

Comment:
```suggestion
        embed=Embed.from_dict(
```
You should [generally leave space](https://legacy.python.org/dev/peps/pep-0008/#other-recommendations) around `=`, but not from within a function call.